modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,246 @@
++
+.Example input config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-vm-latency-checkup
+  namespace: kiagnose
+data:
+  spec.image: registry.redhat.io/container-native-virtualization/vm-network-latency-checkup:v4.11.0
+  spec.timeout: 10m
+  spec.clusterRoles: |
+    kubevirt-vmis-manager
+  spec.param.network_attachment_definition_namespace: "default" <1>
+  spec.param.network_attachment_definition_name: "bridge-network" <2>
+  spec.param.max_desired_latency_milliseconds: "10" <3>
+  spec.param.sample_duration_seconds: "5" <4>
+----
+<1> The namespace where the `NetworkAttachmentDefinition` object resides.
+<2> The name of the `NetworkAttachmentDefinition` object.
+<3> Optional: The maximum desired latency, in milliseconds, between the virtual machines. If the measured latency exceeds this value, the check fails.
+<4> Optional: The duration of the latency check, in seconds.
+
+. Create the config map in the framework’s namespace:
++
+[source,terminal]
+----
+$ oc apply -f <latency-config-map>.yaml
+----
+
+. Create a `Job` object to run the checkup:
++
+.Example job manifest
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kubevirt-vm-latency-checkup
+  namespace: kiagnose
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      serviceAccount: kiagnose
+      restartPolicy: Never
+      containers:
+        - name: framework
+          image: registry.redhat.io/container-native-virtualization/checkup-framework:v4.11.0
+          env:
+            - name: CONFIGMAP_NAMESPACE
+              value: kiagnose
+            - name: CONFIGMAP_NAME
+              value: kubevirt-vm-latency-checkup
+----
+
+. Apply the `Job` manifest. The checkup uses the ping utility to verify connectivity and measure latency.
++
+[source,terminal]
+----
+$ oc apply -f <latency-job>.yaml
+----
+
+. Wait for the job to complete:
++
+[source,terminal]
+----
+$ oc wait --for=condition=complete --timeout=10m job.batch/kubevirt-vm-latency-checkup -n kiagnose
+----
+
+. Review the results of the latency checkup by retrieving the status of the `ConfigMap` object. If the measured latency is greater than the value of the `spec.param.max_desired_latency_milliseconds` attribute, the checkup fails and returns an error.
++
+[source,terminal]
+----
+$ oc get configmap kubevirt-vm-latency-checkup -n kiagnose -o yaml
+----
++
+.Example output config map (success)
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-vm-latency-checkup
+  namespace: kiagnose
+...
+  status.succeeded: "true"
+  status.failureReason: ""
+  status.result.minLatencyNanoSec: 2000
+  status.result.maxLatencyNanoSec: 3000
+  status.result.avgLatencyNanoSec: 2500
+  status.results.measurementDurationSec: 300
+...
+----
+
+. Delete all the framework resources that you previously created:
```

Comment:
Can you have a link to `Example framework manifest file` ? I am not sure the reader is able to understand easily to which file we refer here.
And if we think about it, also the `Example checkup role manifest file` needs to be removed.

A note can say that these resource do not have to be deleted if there is an intention to run the checkup again.